content/docs/guides/ads_on_amp/ads_getting_started.md

```diff
@@ -0,0 +1,63 @@
+---
+$title : "Getting Started"
+$order : 0
+$parent: /content/docs/guides/ads_on_amp
+---
+
+## Three Easy Steps to Serving Ads on Your AMP Page
+
+Not sure how to get started? In this tutorial, you’ll learn how to quickly and easily serve ads on your AMP page.
```

Comment:
Is this a tutorial or a guide?  Previously you indicated "Many of the best practices in this guide".

Could change to "In this short guide, you'll ..."